frontend/src/store/facilityBooking/action.ts

```diff
@@ -288,6 +288,13 @@ export const resetNewBooking = () => (dispatch: Dispatch<ActionTypes>) => {
   })
 }
 
+/**
+ * Takes in `bookingId` and updates `selectedBooking` for the `EditBooking` and `ViewBooking` pages.
+ * @param bookingId (number)
+ * @returns updates `selectedBooking`
```

Comment:
```suggestion
 * @returns updates `selectedBooking`, `isLoading`
```

Reply:
if you realised that there is a `dispatch(setIsLoading(false))` in this function which updates `isLoading` haha